components/Snackbar/src/MDCSnackbarManager.m

```diff
@@ -246,7 +246,7 @@ - (void)displaySnackbarViewForMessage:(MDCSnackbarMessage *)message {
             completion:^{
               if (snackbarView.accessibilityViewIsModal || message.focusOnShow ||
                   ![self isSnackbarTransient:snackbarView]) {
-                UIAccessibilityPostNotification(UIAccessibilityLayoutChangedNotification,
+                UIAccessibilityPostNotification(UIAccessibilityScreenChangedNotification,
```

Comment:
I would prefer us not to make this change. If we really want some clients to be able to vocalize the accessibility hint on the first appearance of the Snackbar, we should probably provide an API to set the notification type.  In general, Snackbars aren't considered a "major portion of the screeen" being taken-up by the new view.  

I also agree with Cody that something like this should be opt-in rather than breaking client behavior.

Reply:
Snackbar already uses the notion of UIAccessibilityScreenChangedNotification in its code. I believe the intended behavior is for the hint to be read, so i respectfully disagree.